### ISSUE TITLE
Patch required to start the game

### ISSUE THREAD

**Issue by Shaddow1974** [dalaranwow/dalaran-wow#4754]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I Tried to launch the game for the first time, set realm list is correct; double checked. The launch screen with title and maintenance info comes up, I entered my username and pass, the game starts to load and then notifies me that a patch is required, but not which version, otherwise I would try to manually patch via download of current launcher. Anyway, then the message pops up and says click restart to install patch. I followed these instructions and every time it says" failed to restart, contact support or reinstall".
**Expected behaviour**: Tell us what should happen instead
The game should load and I should be playing...
**Steps to reproduce the problem**:

1. Download 3.3.5
2. Sign in
3. Click restart when prompted

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by fsironman** [comment]
try to really use your username and not your email

**Comment by ashido96** [comment]
Download the launcher from the dalaran wow website instead of somewhere else.

**Comment by Shaddow1974** [comment]
Kyle,  I did download the client from the Dalaran website. I figured out what it was that I was doing wrong anyway. The problem was I was entering in my registered email address instead of my username along with my password. For some reason when you do this it tells you the game needs to be patched some kind of strange bug I guess. Thanks for your response  Jamie-


Sent via the Samsung Galaxy S7 edge, an AT&T 4G LTE smartphone
-------- Original message --------From: Kyle Stulen <notifications@github.com> Date: 6/20/17  1:33 PM  (GMT-05:00) To: dalaranwow/dalaran-wow <dalaran-wow@noreply.github.com> Cc: Shaddow1974 <JmeEastman@Yahoo.com>, Author <author@noreply.github.com> Subject: Re: [dalaranwow/dalaran-wow] Patch required to start the game (#4754) 
Download the launcher from the dalaran wow website instead of somewhere else.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/dalaranwow/dalaran-wow","title":"dalaranwow/dalaran-wow","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/dalaranwow/dalaran-wow"}},"updates":{"snippets":[{"icon":"PERSON","message":"@ashido96 in #4754: Download the launcher from the dalaran wow website instead of somewhere else. "}],"action":{"name":"View Issue","url":"https://github.com/dalaranwow/dalaran-wow/issues/4754#issuecomment-309831715"}}}

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
